extension/src/format/sexpression.ts

```diff
@@ -892,96 +1159,7 @@ export class Sexpression extends LispAtom {
-
-    getRange(){
-        const begin: LispAtom = this.atoms[0];
-        const close: LispAtom = this.atoms[this.atoms.length -1];
-        return new Range(begin.line, begin.column, close.line, (close.column + close.symbol.length));
-    }
-
-    
-    contains(position: Position): boolean {
-        return this.getRange().contains(position);
-    }
-
-
-    // This version was necessary to properly alternate over SETQ Name vs Value 
-    private isValidForSetq(atom: LispAtom) {
-        return !atom.isComment() && !['\'', '(', ')', '.'].includes(atom.symbol) && (atom instanceof Sexpression || atom.symbol.trim().length > 0);
-    }    
-    // This is general purpose utility to make sure primitives such as strings, numbers and decorations are not evaluated
-    private notNumberStringOrProtected(atom: LispAtom) {        
-        return !atom.isComment() && !['\'', '(', ')', '.'].includes(atom.symbol) && !(/^".*"$/.test(atom.symbol)) && !(/^\'{0,1}\-{0,1}\d+[eE\-]{0,2}\d*$/.test(atom.symbol));
-    }
-
-
-    // This is an ittoration helper that makes sure only useful LispAtom/Sexpression's get used for contextual data collection purposes.
-    nextKeyIndex(currentIndex: number, forSetq?: boolean): number {
-        let index = currentIndex + 1;
-        if (index < this.atoms.length) {
-            const atom = this.atoms[index];
-            const flag = forSetq ? this.isValidForSetq(atom) : this.notNumberStringOrProtected(atom);
-            if (atom instanceof LispAtom && flag){
-                return index;               
-            } else {
-                return this.nextKeyIndex(index, forSetq);
-            }
-        } else {
-            return -1;
-        }
-    }
-
-    getNthKeyAtom(significantNth: number){
-        let num = 0;        
-        for (let i = 0; i <= significantNth; i++) {
-            num = this.nextKeyIndex(num, true);
-        }          
-        if (num < this.atoms.length && num >= 0){
-            return this.atoms[num];
-        } else {
-            return null;
-        }
-    }
-
-
-    // Used a lot like a DOM selector to navigate/extract values from Sexpresions
-    findChildren(regx: RegExp, all: boolean): Sexpression[] {
-        let result: Sexpression[] = [];       
-        let index = this.nextKeyIndex(0);
-        if (!this.atoms[index]){
-            return result;
-        } else if (!(this.atoms[index] instanceof Sexpression) && regx.test(this.atoms[index].symbol) === true){
-            result.push(this);
-            if (all === true) {
-                this.atoms.filter(f => f instanceof Sexpression).forEach((atom: Sexpression) => {
-                    result = result.concat(atom.findChildren(regx, all));
-                });
-            }
-        } else {
-            this.atoms.filter(f => f instanceof Sexpression).forEach((atom: Sexpression) => {
-                result = result.concat(atom.findChildren(regx, all));
-            });
-        }
-        return result;
-    }
```

Comment:
All of this was moved into the LispContainer because it had nothing to do with and/or was not being used by formatting context.